translations/README/PERSIAN.md

```diff
@@ -0,0 +1,384 @@
+
+### این پروژه برای چه کسانی است؟
+
+- این پروژه برای افراد کاملا مبتدی است. اگر بلدید که تگ `<a href="" target=""></a>` بنویسید، پس شما از پسش بر میاید.
+- همچنین این پروژه برای افرادی است که تجربه بیشتری دارند ولی میخواهند که شروع به مشارکت کنند و اعتماد بنفس و تجربیشان در فضای مشارکتی را بیشتر کنند.
+
+### چرا نیاز دارم تا اینکارو انجام بدم؟
+
+هر توسعه دهنده وبی، بدون در نظر گرفتن تجربش، نیاز داره تا بلد باشه چطور میتونه از گیت استفاده کنه، و گیتهاب معروف ترین سرویس هاست گیت هست که همه ازش استفاده میکنند. همچنین قلب جامعه متن باز هست. به راحتی کار کردن با گیتهاب توانایی مهمیه و مشارکت در یک پروژه باعث افزایش اعتماد بنفستون میشه و به شما امکاناتی میده که میتونید ازش در پروفایلتون استفاده کنید.
+
+اگر شما تازه اول راه هستید و براتون سوال هست که چرا نیاز دارید تا گیت و گیتهاب را یاد بگیرید، جواب شما اینجاست : [شما باید دیروز گیت را یاد میگرفتید](https://codeburst.io/number-one-piece-of-advice-for-new-developers-ddd08abc8bfa 'New Developer? You should’ve learned Git yesterday. by Brandon Morelli, creator of CodeBurst.io').
+
+### قراره چه مشارکتی داشته باشم؟
+
+![Contributor Card](/readme-only/card.PNG 'Contributor Card')
+
+شما قراره در ساختن این کارت مشارکت کنید ([وبسایت پروژه](https://syknapse.github.io/Contribute-To-This-Project/ 'https://syknapse.github.io/Contribute-To-This-Project')). این کارت شامل اسم شما، آیدی توییتر شما، خلاصه کوتاهی از خودتان و 3 لینک مفید برای توسعه دادن وب که شما پیشنهاد میکنید است.
+
+شما قرار است تمپلیت این کارت درون فایل `HTML` را کپی کنید و با اطلاعات خودتان تکمیلش کنید.
+
+---
+
+### ترجمه‌ها
+
+این آموزش به [زبان های دیگر](/translations/README.md) نیز در دسترس است.
+
+|     [Arabic (عربي)](/translations/README/ARABIC.md)     |  [Bangla (বাংলা)](/translations/README/BANGLA.md)  | [Chinese (Traditional) (繁體中文)](/translations/README/CHINESE_TRADITIONAL.md) | [Dutch](/translations/README/DUTCH.md) |            [English (English)](/README.md)             |  
+| :---------------------------------------------: | :---------------------------------------: | :-----------------------------------------: | :---------------------------------------: | :---------------------------------------: |
+| [French (Français)](/translations/README/FRENCH.md) |  [German (Deutsch)](/translations/README/GERMAN.md)  |      [Hindi (हिंदी)](/translations/README/HINDI.md)      | [Italian (Italiano)](/translations/README/ITALIAN.md) | [Japanese (日本語)](/translations/README/JAPANESE.md) |
+  [Korean (한국어)](/translations/README/KOREAN.md)  | [Polish (Polski)](/translations/README/POLISH.md) | [Portuguese (Portuguese)](/translations/README/PORTUGUESE.md) | [Russian (Русский)](/translations/README/RUSSIAN.md) |  [Serbian (Српски)](/translations/README/SERBIAN.md)  |
+  [Spanish (Español)](/translations/README/SPANISH.md) | [Turkish (Türkçe)](/translations/README/TURKISH.md) | [Ukrainian (українська)](/translations/README/UKRAINIAN.md) | [Norwegian (Norsk)](/translations/README/NORWEGIAN.md) | [Persian (فارسی)](/translations/README/FARSI.md)
```

Comment:
the link in Persian language should be `/translation/README/PERSIAN.md` according to the file name. also don't put it here, as it is the Persian translation file, you need to fix the link in other files where you've mentioned this translation.